src/LiveDevelopment/LiveDevelopment.js

```diff
@@ -195,6 +197,10 @@ define(function LiveDevelopment(require, exports, module) {
      * Handles of registered servers
      */
     var _regServers = [];
+    
+    PreferencesManager.definePreference("livedev.wsPort", "number", 8125, {
+        description: Strings.DESCRIPTION_LIVEDEV_WEBSOCKET_PORT
+    });
```

Comment:
Since you are doing for LiveDevelopment, maybe you should do the same for MultiBrowserLivePreview too. See also #11957